docs/prow/manage-component-jobs-with-templates.md

```diff
@@ -12,124 +12,164 @@ This document describes how to define, modify, and remove Prow jobs for Kyma com
+   See the example:
+   
+   ```go
+   ...
+   {path: "apiserver-proxy", image: tester.ImageGolangBuildpack1_12},
+   {path: "apiserver-proxy", image: tester.ImageBootstrap20181204, suite: tester.NewGenericComponentSuite,
+     additionalOptions: []jobsuite.Option{
+       jobsuite.JobFileSuffix("generic"),
+       jobsuite.Since(releases.Release17),
+       jobsuite.Optional(),
+     },
+   },
+   ```
+   
+   Same as with component jobs, mark the component test as optional at this stage by adding the `jobsuite.Optional()` entry.
+   
+   If you have access to a Prow cluster, you can test a Prow job on it. For details, see the [official documentation](https://github.com/kubernetes/test-infra/blob/master/prow/build_test_update.md#how-to-test-a-prowjob).
+   
+   When writing tests for a new component, use the `tester.GetKymaReleasesSince({next release})` function to create tests for release jobs.
 
-When writing tests for a new component, use the `tester.GetKymaReleasesSince({next release})` function to create tests for release jobs.
 
 3. Generate jobs.
 
-Run this command to generate jobs previously defined in the `config.yaml` file:
+   Run one of these commands to generate jobs previously defined in the `config.yaml` file:
 
-```bash
-go run development/tools/cmd/rendertemplates/main.go --config templates/config.yaml
-```
+   ```bash
+   go run development/tools/cmd/rendertemplates/main.go --data path/to/directory/with/data/files
+   ```
+   or
+   ```bash
+   make jobs-definitions
+   ```
+   
+   As a result, the Render Templates tool generates the requested job files.
 
-As a result, the Render Templates tool generates the requested job files.
 
 4. Check your configuration locally.
 
-Use the `development/validate-config.sh` script to validate your Prow configuration. The script accepts three arguments:
+   Use the `development/validate-config.sh` script to validate your Prow configuration. The script accepts three arguments:
 
-- The path to the plugins configuration file (`prow/plugins.yaml`)
-- The path to the generic configuration file (`prow/config.yaml`)
-- The path to the directory with job definitions (`prow/jobs/`)
+   - The path to the plugins configuration file (`prow/plugins.yaml`)
+   - The path to the generic configuration file (`prow/config.yaml`)
+   - The path to the directory with job definitions (`prow/jobs/`)
 
-See an example:
+   See an example:
 
-```bash
-cd $GOPATH/src/github.com/kyma-project/test-infra
-./development/validate-config.sh prow/plugins.yaml prow/config.yaml prow/jobs/
-```
+   ```bash
+   cd $GOPATH/src/github.com/kyma-project/test-infra
+   ./development/validate-config.sh prow/plugins.yaml prow/config.yaml prow/jobs/
+   ```
 
 5. Merge the changes.
 
-Create a PR with your changes in the `config.yaml` file and the job files generated by the Render Templates.
+   Create a PR with your changes in the `config.yaml` file and the job files generated by the Render Templates.
```

Comment:
...by the templates or by the Render Templates tool?

Reply:
...by Render Templates tool from template 😅 but I need to change this sentence anyway, cause it's not `config.yaml` anymore